packages/lib-utils/src/components/table/VirtualizedTable.tsx

```diff
@@ -167,20 +168,20 @@ const VirtualizedTable: React.FC<VirtualizedTableProps<AnyObject>> = ({
       CustomNoDataEmptyState={CustomNoDataEmptyState}
     >
       <div role="grid" aria-label={ariaLabel} aria-rowcount={data?.length || 0}>
-        <TableComposable aria-label={ariaLabel} role="presentation">
+        <TableComposable aria-label={ariaLabel} role="presentation" gridBreakPoint="">
```

Comment:
Out of curiosity, what does setting `gridBreakPoint=""` fix? Will this change cause issues with the table wrapping in smaller browser viewports?

Reply:
That's again an issue with compatibility of the virtualized extension with table composable since when it switches to mobile view, we would miss the table header:
![image](https://user-images.githubusercontent.com/50696716/181465837-93ad7dcf-a007-4640-a36b-2a9fc6636bfa.png)

So I disabled it:
![image](https://user-images.githubusercontent.com/50696716/181466135-9501f17c-ec56-4380-97ce-dee9add6db69.png)